cms-xcache/xrootd/xcache.cfg

```diff
@@ -0,0 +1,2 @@
+continue /etc/xrootd/config-common.d
+continue /etc/xrootd/config-xrootd.d
```

Comment:
`continue` is a one-shot command.  Only the first mention is parsed and the remaining of the configuration file is ignored.  I don't think this can work.